src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -550,6 +550,44 @@ bool CommandRequestImpl::CheckAllowedParameters() {
   return true;
 }
 
+bool CommandRequestImpl::CheckHMICapabilities(
+    mobile_apis::ButtonName::eType button) const {
```

Comment:
@LuxoftAKutsan yes, added `const` in https://github.com/smartdevicelink/sdl_core/pull/1621/commits/e5fb4510b64635d824bbb48a205b7c3e4ed0e92c